contracts/modules/royalty/RoyaltyModule.sol

```diff
@@ -50,8 +57,15 @@ contract RoyaltyModule is
     string public constant override name = ROYALTY_MODULE_KEY;
 
     /// @notice Constructor
+    /// @param disputeModule The address of the dispute module
+    /// @param licenseRegistry The address of the license registry
     /// @custom:oz-upgrades-unsafe-allow constructor
-    constructor() {
+    constructor(address disputeModule, address licenseRegistry) {
```

Comment:
Do you need to pass in the address or retrieve it from module registry? I thought @kingster-will has a better approach?

Reply:
It seems to be the same logic as in the LicensingModule.sol - also passes the addresses in constructor.